### DISCUSSION TITLE
how to create straight or rectangular arrows within mindmaps

### DISCUSSION THREAD

**Discussion by nnako** [plantuml/plantuml#1017] [Q&A]
I'd like to create straight or rectangular arrows between my mindmap elements.

By default, lines appear to connect mindmap elements in a "curvy" way. According to plantuml's [mindmap docs](https://plantuml.com/de/mindmap-diagram), I can generally adjust the arrow style like this (there are styling options for line style, line thickness and line color):

```
<style>
arrow {
    LineStyle 4
    LineThickness 0.5
    LineColor green
}
</style>
```

then define a normal mindmap like this:

```
* Hi =)
** sometimes i have node in wich i want to write a long text
*** this results in really huge diagram
**** of course, i can explicit split with a\nnew line
**** but it could be cool if PlantUML was able to split long lines, maybe with an option 
```

If there is, where do I find a list of all available arrow styling options? otherwise, where would I look up the mindmap's line style options within the code base?

Thanks.

**Comment by The-Lum** [comment]
It is not yet implemented:
- For rectangular arrow, see WBS:
  - https://plantuml.com/wbs-diagram
- And about `style`, see doc. here:
  - https://plantuml.com/style-evolution
  - https://plantuml.com/en/style-evolution#faebbd3c7bf835b5

_If that can help,_
Regards.
